roles/docker/tasks/main.yml

```diff
@@ -36,3 +36,42 @@
   when:
     - l_use_crio
     - openshift_docker_is_node_or_master | bool
+
```

Comment:
How about:

```yaml
- stat:
    path="{{ docker_default_storage_path }}"
  register: l_docker_storage_link_check

- when:
    - l_use_crio
    - l_docker_storage_link_check.stat.islink is defined and l_docker_storage_link_check.stat.islink
....
```

Reply:
added

Reply:
should the `_docker_storage_link_check.stat.islink is defined and l_docker_storage_link_check.stat.islink` condition be negated?  The link must be created only when it is not already a link

It will also make the code idempotent as noted by @twiest

Reply:
@giuseppe I think you are correct. That was my mistake. 😞